scripts/dockerfiles/polkadot/docker-compose-local.yml

```diff
@@ -1,35 +1,39 @@
 version: '3'
 services:
   node_alice:
-    build:
-      context: .
-      dockerfile: polkadot_builder.Dockerfile
     ports:
       - "30333:30333"
       - "9933:9933"
       - "9944:9944"
-    image: chevdor/polkadot:latest
+      - "9615:9615"
+    image: parity/polkadot:latest
     volumes:
       - "polkadot-data-alice:/data"
-    command: polkadot --chain=polkadot-local --alice -d /data --node-key 0000000000000000000000000000000000000000000000000000000000000001
+    command: |
+      --chain=polkadot-local
+      --alice
+      -d /data
+      --node-key 0000000000000000000000000000000000000000000000000000000000000001
     networks:
       testing_net:
         ipv4_address: 172.28.1.1
 
   node_bob:
-    build:
-      context: .
-      dockerfile: polkadot_builder.Dockerfile
     ports:
-      - "30344:30344"
-      - "9935:9935"
-      - "9945:9945"
-    image: chevdor/polkadot:latest
+      - "30344:30333"
+      - "9935:9933"
+      - "9945:9944"
+      - "29615:9615"
+    image: parity/polkadot:latest
     volumes:
       - "polkadot-data-bob:/data"
     links:
       - "node_alice:alice"
-    command: polkadot --chain=polkadot-local --bob -d /data --port 30344 --rpc-port 9935 --ws-port 9945 --bootnodes '/ip4/172.28.1.1/tcp/30333/p2p/QmRpheLN4JWdAnY7HGJfWFNbfkQCb6tFf4vvA6hgjMZKrR'
+    command: |
+      --chain=polkadot-local
+      --bob
+      -d /data
+      --bootnodes '/ip4/172.28.1.1/tcp/30333/p2p/QmRpheLN4JWdAnY7HGJfWFNbfkQCb6tFf4vvA6hgjMZKrR'
```

Comment:
does it assume the binary?

Reply:
there is an [entrypoint](https://github.com/paritytech/polkadot/blob/be7dbb2c036d0ba6a979ac22f6b4b3f352ad9fb3/scripts/dockerfiles/polkadot/polkadot_builder.Dockerfile#L36)